- add close(2) includes to fix the build
- replace memfd_create() with shm_open(2), OpenBSD lacks the former

Index: Telegram/lib_webview/webview/webview_data_stream_memory.cpp
--- Telegram/lib_webview/webview/webview_data_stream_memory.cpp.orig
+++ Telegram/lib_webview/webview/webview_data_stream_memory.cpp
@@ -8,6 +8,8 @@
 
 #if !defined Q_OS_WIN && !defined Q_OS_MAC
 #include <sys/mman.h>
+#include <fcntl.h>
+#include <unistd.h>
 #endif // !Q_OS_WIN && !Q_OS_MAC
 
 namespace Webview {
@@ -18,7 +20,7 @@ DataStreamFromMemory::DataStreamFromMemory(
 : _data(data)
 , _mime(mime) {
 #if !defined Q_OS_WIN && !defined Q_OS_MAC
-	const auto handle = memfd_create("webview-data-stream", MFD_CLOEXEC);
+	const auto handle = shm_open("webview-data-stream", O_RDWR|O_CLOEXEC, 0600);
 	if (handle == -1) {
 		return;
 	}
